krausening-python/pyproject.toml

```diff
@@ -10,8 +10,8 @@ repository = "https://github.com/TechnologyBrewery/krausening"
 keywords = ["properties", "configuration-management"]
 
 [tool.poetry.dependencies]
-python = "^3.9"
-javaproperties = ">=0.8.1"
+python = ">=3.8"
+javaproperties = [{version = ">=0.8.1", python = ">=3.8,<4.0"}]
```

Comment:
C: Poetry uses very strict matching with a project's dependencies' Python versions. If a dependency's Python requirement doesn’t match the **whole range** specified in the `pyproject.toml`, Poetry will fail to install the dependency. The solution to this problem is to define which versions of Python a particular package can use. In this case, `javaproperties`'s allowed versions only allow Python versions `<4.0`, so we must specify that here to be able to install it.

Reply:
C: Addresses Issue #29